### PR TITLE
chmod: update Dutch translation

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.nl/common/chmod.md [hunk 1]
```diff
@@ -5,28 +5,32 @@
 
 - Geef een gebruiker ([u]ser) die het bestand beheert het recht om deze uit te voeren (e[x]ecute):
 
-`chmod u+x {{bestand}}`
+`chmod u+x {{pad/naar/bestand}}`
 
 - Geef de gebruiker ([u]ser) het recht om een bestand of map te lezen ([r]ead) en schrijven ([w]rite):
 
-`chmod u+rw {{bestand_of_map}}`
+`chmod u+rw {{pad/naar/bestand_of_map}}`
```
Comment (blueskyson) [inline pages.nl/common/chmod.md:12]: ref: [common-arguments.md](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)
```suggestion
`chmod u+rw {{pad/naar/bestand_of_directory}}`
```

File: pages.nl/common/chmod.md [hunk 2]
```diff
@@ -5,28 +5,32 @@
...
-`chmod o= {{bestand}}`
+`chmod o= {{pad/naar/bestand}}`
 
 - Verander de toestemmingen recursief, waarbij de [g]roep en anderen ([o]thers) de mogelijkheid tot schrijven ([w]rite) krijgen:
 
 `chmod -R g+w,o+w {{map}}`
+
+- Geef recursief alle gebruikers ([a]ll users) toegang om bestanden te lezen ([r]ead) en uitvoertoestemming (e[X]ecute) voor alle onderliggende mappen in een map:
+
+`chmod -R a+rX {{pad/naar/map}}`
```
Comment (blueskyson) [inline pages.nl/common/chmod.md:36]:
```suggestion
`chmod -R a+rX {{pad/naar/directory}}`
```

File: pages.nl/common/chmod.md (hunk 1 above)
Comment (sebastiaanspeck) [reply pages.nl/common/chmod.md:12]: Directory in Dutch is map.

File: pages.nl/common/chmod.md (hunk 2 above)
Comment (sebastiaanspeck) [reply pages.nl/common/chmod.md:36]: Directory in Dutch is map.

**Comment by sebastiaanspeck** [comment]
I have opened a PR to rename the common-arguments to map. If that one is merged, I will open a bulk-PR to rename pad/naar/directory -> pad/naar/map and pad/naar/bestand_of_directory -> pad/naar/bestand_of_map. Or search for all occurrences of directory in pages.nl 💪🏼

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by kbdharun — COMMENTED** [review]
Now that the common arguments PR got merged, can you revert the directory word to its native equivalent?

**Comment by sebastiaanspeck** [comment]
Will be picked up in https://github.com/tldr-pages/tldr/pull/11030

**Pull request merged**